### ISSUE TITLE
mock build fails with ghc-9.6

### ISSUE THREAD

**Issue by juhp** [luntain/error-or-bundle#4]
```
       Building library for mock-time-0.1.0..                                                                                       
       [5 of 5] Compiling Control.Monad.Trans.SimulatedTime                                                                         
                                                                                                                                    
       /var/stackage/work/unpack-dir/unpacked/mock-time-0.1.0-34fa4e5740f526b54dce99c1d19858d5833663073305b9075fc931578612b9e8/src/C
ontrol/Monad/Trans/SimulatedTime.hs:58:7: error: [GHC-76037]      
           Not in scope: type constructor or class ‘MonadPlus’                                                                      
          |                                                                                                                         
       58 |       MonadPlus,                                      
          |       ^^^^^^^^^                                       
                                                                                                                                    
       /var/stackage/work/unpack-dir/unpacked/mock-time-0.1.0-34fa4e5740f526b54dce99c1d19858d5833663073305b9075fc931578612b9e8/src/C
ontrol/Monad/Trans/SimulatedTime.hs:59:7: error: [GHC-76037]                                                                        
           Not in scope: type constructor or class ‘MonadFix’                                                                       
           Suggested fix:                                         
             Perhaps use one of these:                                                                                              
               ‘MonadZip’ (imported from Control.Monad.Zip),                                                                        
               ‘MonadFail’ (imported from Prelude)  
          |                                                                                                                         
       59 |       MonadFix,                                                                                                         
          |       ^^^^^^^^
                                                                                                                                    
       /var/stackage/work/unpack-dir/unpacked/mock-time-0.1.0-34fa4e5740f526b54dce99c1d19858d5833663073305b9075fc931578612b9e8/src/C
ontrol/Monad/Trans/SimulatedTime.hs:123:7: error: [GHC-76037]
           Not in scope: type constructor or class ‘MonadPlus’
           |                                                                                                                        
       123 |       MonadPlus,
           |       ^^^^^^^^^
        
       /var/stackage/work/unpack-dir/unpacked/mock-time-0.1.0-34fa4e5740f526b54dce99c1d19858d5833663073305b9075fc931578612b9e8/src/C
ontrol/Monad/Trans/SimulatedTime.hs:124:7: error: [GHC-76037]
           Not in scope: type constructor or class ‘MonadFix’
           Suggested fix:
             Perhaps use one of these:
               ‘MonadZip’ (imported from Control.Monad.Zip),
               ‘MonadFail’ (imported from Prelude)
           |
       124 |       MonadFix,
           |       ^^^^^^^^
```

**Comment by luntain** [comment]
Fixed in master

**Comment by luntain** [comment]
fixed in mock-time-0.1.1
